ci/docker-compose.yml

```diff
@@ -84,6 +84,12 @@ services:
       - KEYCLOAK_APP_USER=alice
       - KEYCLOAK_APP_USER_PASSWORD=alice
       - KEYCLOAK_APP_USER_EMAIL=alice@conjur.net
+      - KEYCLOAK_SECOND_APP_USER=bob
```

Comment:
Is there some reason these user credentials need to be defined in the container env? It looks like we're just using the container env as a way to temporarily store the credentials so that they can be extracted and used in the bash script that calls `create_user` later on but maybe I'm missing something.

Reply:
yes, we want one place to have all the keycloak variables. these variables are used for `create_user` both in `ci/test` & `dev/start` so we put it here to have 1 source of truth.

This is relevant for `KEYCLOAK_APP_USER, KEYCLOAK_SECOND_APP_USER, KEYCLOAK_NON_CONJUR_APP_USER` and all the corresponding vars (password, email).

Do you thinhk it's better to have them initialized in `ci/test` & `dev/start` instead?

Reply:
Ah okay. Nah this is fine.